src/material/checkbox/checkbox.html

```diff
@@ -2,7 +2,7 @@
   <span class="mat-checkbox-inner-container"
        [class.mat-checkbox-inner-container-no-side-margin]="!checkboxLabel.textContent || !checkboxLabel.textContent.trim()">
     <input #input
```

Comment:
Why not use `.mdc-radio__native-control` here? That's how it looks like MDC does this.  (they also set opacity zero)

Reply:
That class is where I got the inspiration for these styles (especially `opacity: 0`). However, I don't think that we want to directly use the MDC styles in the non-MDC-based components?

Reply:
Ah, I missed that this wasn't touching the MDC one. Fixing the MDC components here is the main priority, with fixing the pre-MDC ones being secondary.

Reply:
Understood. I can't reproduce any hover text issues with the MDC components, as mentioned in the issue (with videos).